src/integration-test/java/io/aiven/kafka/connect/http/AvroIntegrationTest.java

```diff
@@ -179,16 +175,12 @@ final void testBasicDelivery() throws ExecutionException, InterruptedException {
         for (final Future<RecordMetadata> sendFuture : sendFutures) {
             sendFuture.get();
         }
-        TestUtils.waitForCondition(() -> bodyRecorderHandler.recorderBodies().size() >= expectedBodies.size(),
-                15000,
-                "All requests received by HTTP server"
-        );
-        log.info("Recorded request bodies: {}", bodyRecorderHandler.recorderBodies());
```

Comment:
I suppose this was here for debugging of the test, why remove it? Does the `assertThat` utility provide a better message in case of failure?

Reply:
Yes, AssertJ and it's `assertThat` provide a lot more clear messages. Also in this case I checked how this one is printed and it is lost among other logs and not really visible in case of failure.